open-craft/unit-xblock
# Initial implementation of a Unit XBlock [LX-104] [SE-437]

Phew, the amount of boilerplate needed to get this XBlock with < 100 lines of code set up with most of the correct tooling is _ridiculous_!

# What it is

A minimal, pure-XBlock substitute for `vertical`. (`vertical` has a _lot_ of tech debt and mixins that can make development of new runtimes etc. messier than necessary, e.g. it has totally different XML parsing behavior than XBlocks do, because it uses XModule XML parsing code.)

# Test instructions:

You can run `make dev.run` then go to http://localhost:8000 to see it. Tests run in CircleCI and cover python 2 and 3.
